Cargo.toml

```diff
@@ -51,6 +45,4 @@ tracing = { version = "^0.1.34", features = ["attributes"] }
 tracing-subscriber = "^0.3.11"
 url = "^2.2.2"
 uuid = "^1.1.2"
-
-
-
+sspi = "0.11.0"
```

Comment:
Should we make it an optional feature?

Reply:
Yes, I think so. I think most users of the crate won't need it, so feature gating this is probably wise.